model/device.go

```diff
@@ -93,6 +94,138 @@ const (
 	DevTypeTest       = "Test"
 )
 
+// Consts containing defaults for all device configs.
+const (
+	defaultActPeriod int64 = 60
+	defaultMonPeriod int64 = 60
+)
+
+// Consts containing defaults for a controller config.
+const (
+	defaultControllerInputs  string = "A0,X50,X51,X60,X10"
+	defaultControllerOutputs string = "D14,D15,D16" // Peripheral Power.
+)
+
+// controllerDefaultVars is a map which relates variable names to their values for
+// the default values of controller variables.
+var controllerDefaultVars = map[string]string{
+	"AlarmNetwork":         "10",
+	"AlarmPeriod":          "5",     // 5 seconds.
+	"AlarmRecoveryVoltage": "840",   // 25.4545 V.
+	"AlarmVoltage":         "825",   // 25 V.
+	"AutoRestart":          "600",   // 10 minutes.
+	"Power":                "false", // initialise to OFF.
+	"Pulses":               "3",     // 3 pulses per cycle.
+	"PulseWidth":           "2",     // 1s ON, 1s OFF.
+	"PulseCycle":           "30",    // 30s between flashes.
+	"Suppress":             "false",
+}
+
+// controllerDefaultSensors contains a slice of the default sensors for a controller device.
+const controllerDefaultSensors = []model.SensorV2{
+	{Name: "Battery Voltage", Pin: "A0", Quantity: "DCV", Func: "scale", Args: "0.0289", Units: "V", Format: "round1"},
+	{Name: "Analog Value", Pin: "X10", Quantity: "OTH", Func: "none", Format: "round1"},
+	{Name: "Air Temperature", Pin: "X50", Quantity: "MWH", Func: "linear", Args: "0.1,-273.15", Units: "C", Format: "round1"},
+	{Name: "Humidity", Pin: "X51", Quantity: "MMB", Func: "scale", Args: "0.1", Units: "%", Format: "round2"},
+	{Name: "Water Temperature", Pin: "X60", Quantity: "MTW", Func: "linear", Args: "0.1,-273.15", Units: "C", Format: "round1"},
+}
+
+// controllerDefaultActs contains a slice of the default actuators for a controller device.
+const controllerDefaultActs = []model.ActuatorV2{
+	{Name: "Empty 1", Var: "Power1", Pin: "D16"},
```

Comment:
Please make sure you address the comment from the previous PR too. Alan suggested that these be named Power 1, Power 2 etc. instead of Empty 1, Empty 2 etc. which I think are better default names too.

Reply:
I was using empty to indicate clearly that the power had nothing attached to it, and that it should probably be updated once a device has been connected to it. If it is labeled Power xxx, a user will likely not rename it, and then forget which device is connected to which power.

I think calling it something like empty by default is a good way to encourage users to rename them once they connect a peripheral.

Reply:
Suggest naming them "Device 1", "Device 2", so users know they correspond to devices.